temporian/core/event_set_ops.py

```diff
@@ -4382,3 +4382,46 @@ def drop(
         from temporian.core.operators.select import drop
 
         return drop(self, feature_names=feature_names)
+
+    def assign(
+        self: EventSetOrNode, **others: EventSetOrNode
+    ) -> EventSetOrNode:
+        """Assign new features to an EventSet
```

Comment:
missing fullstop, missing crosslink to EventSet ([`EventSet`][temporian.EventSet])